composables/useApi/useApi.test.ts

```diff
@@ -54,6 +55,8 @@ describe('useApi', () => {
         })
 
         expect(onErrorSpy).toBeCalledTimes(1)
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-expect-error
```

Comment:
Looking this instruction up, I'm not sure why we need this here. I don't see why the compiler would think the next line would be an error... 🤔

Reply:
Ya i'm not sure either, We can always go back and fix later as its not causing any errors it seems